src/Microsoft.Graph/Requests/Generated/EducationRootRequest.cs

```diff
@@ -117,9 +117,23 @@ public System.Threading.Tasks.Task<EducationRoot> UpdateAsync(EducationRoot educ
         /// </summary>
         /// <param name="educationRootToUpdate">The EducationRoot to update.</param>
         /// <param name="cancellationToken">The <see cref="CancellationToken"/> for the request.</param>
+		/// <exception cref="ClientException">Thrown when an object returned in a response is used for updating an object in Microsoft Graph.</exception>
```

Comment:
I accepted the generator PR to soon. Please fix the indents to match the existing indents.

Reply:
Blimey! It slipped my mind that I used tabs instead of spacing. I've corrected this.